Library/Formula/zanata-client.rb

```diff
@@ -0,0 +1,18 @@
+class ZanataClient < Formula
+  desc "Zanata translation system command-line client"
+  homepage "http://zanata.org/"
+  url "https://search.maven.org/remotecontent?filepath=org/zanata/zanata-cli/3.7.4/zanata-cli-3.7.4-dist.tar.gz"
+  sha256 "4424322bffb81f5185f87d1f11c7e92cde504c736695dbb4ced5854e49672037"
+
+  depends_on :java => "1.7+"
+
+  def install
+    libexec.install Dir["*"]
+    (bin/"zanata-cli").write_env_script libexec/"bin/zanata-cli", Language::Java.java_home_env
```

Comment:
Thanks, I'll change it.  Out of curiosity, would it be possible for the `Language::Java.java_home_env` bit to just automatically determine that versioning based on the `depends_on` line above?  It feels kind of weird to have to specify the dependency twice.